internal/node/config/config.go

```diff
@@ -169,6 +200,11 @@ func (n *NodeConfig) TailscaleAuthkey() string {
 	return viper.GetString("tailscale_authkey")
 }
 
+func (n *NodeConfig) TailScaleStatePath() string {
+	// Note: this is intentionally not configurable for simplicity's sake.
+	return filepath.Join(n.HabitatPath(), "tailscale_state")
```

Comment:
hm this defaults to `ts_state`

Reply:
Ahh I missed that there was another config for this